wiki/Tournaments/CMT_4K/2018/en.md

```diff
@@ -0,0 +1,170 @@
+# Chinese osu!mania 4K Tournament 2018
+
+![CMT 4K 2018](img/banner.jpg)
+
+The **Chinese osu!mania 4K Tournament 2018** (***CMT 4K 2018***) is the third one-on-one, osu!mania 4K specified Chinese regional tournament, hosted by CMT team.
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration | 2018-09-10/2018-09-29 |
+| Qualification Stage | 2018-10-06/2018-10-07 |
+| Drawings | 2018-10-07 (20:00 UTC+8) |
+| Group Stage | 2018-10-13/2018-10-14 |
+| Round of 16 | 2018-10-20/2018-10-21 |
+| Round of 8 | 2018-10-27/2018-10-28 |
+| Semifinals | 2018-11-03/2018-11-04 |
+| Finals - Week 1 | 2018-11-10/2018-11-11 |
+| Finals - Week 2 | 2018-11-17/2018-11-18 |
+
+## Prizes
+
+All prizes are sponsored by **[Arkaaalin](https://osu.ppy.sh/users/7477382)**.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![](/wiki/shared/GCrown.png) | 8 months of osu!supporter tag, unique userpage badge |
+| ![](/wiki/shared/SCrown.png) | 4 months of osu!supporter tag |
+| ![](/wiki/shared/BCrown.png) | 2 months of osu!supporter tag |
+
+## Staff
+
```

Comment:
The Chinese osu!mania 4K Tournament 2018 is run by various community members by distributing the multitude of tasks into various fields of responsibility.